launchAndRender.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
+
+#run nbconvert on a notebook, copy to the templates folter, create a new template and render.
+
+#First argument: File name of the notebook to work on: e.g., if the notebook is analysis.ipynb, argument is analysis.
+
+cd notebooks/
```

Comment:
This is a good idea, but I think that it would be better to write this code in python instead of in a shell script. 

I think that a good interface could be: 

```
python  render.py  base.html  my_nb_1.ipynb  my_nb_2.ipynb ... 
```

* The user could provide any number of notebooks to insert. 
* In my_output.html, we would use jinja2 tags to specify where each notebook should be inserted, based on the name the notebook
* The script would run jupyter nbconvert (maybe there is a python API, or with os.system or something)
* Finally, the ouput file would be rendered

One could also think about the options. For example, the name of the output file (index.html by default)

Reply:
I'll use this interface, then. I'd have to research jinja2 a bit to see what templating options are available.